src/main/kotlin/racingcar/communication/Pit.kt

```diff
@@ -1,12 +1,13 @@
 package racingcar.communication
 
-import racingcar.domain.cars.NumberOfRacingCars
 import racingcar.communication.input.Input
 import racingcar.communication.output.Output
+import racingcar.domain.cars.RacingCar
 
 object Pit {
-    fun readyToCourseInRacingCars(input: Input, output: Output): NumberOfRacingCars {
-        output.numberOfCarsMessage()
-        return NumberOfRacingCars(input.numberOfRacingCars())
+    fun courseInRacingCars(input: Input, output: Output): List<RacingCar> {
+        output.numberOfCarsNames()
+        val racingCarsNames = input.racingCarsNames().map { it.trim() }
+        return racingCarsNames.map { RacingCar(name = it) }
```

Comment:
```suggestion
        return input
            .racingCarsNames()
            .asSequence()
            .map { it.trim() }
            .map { RacingCar(name = it) }
            .toList()
```
체이닝으로 별도의 변수 없이 활용할 수 있을 것 같아요

Reply:
질문이 있는데요~
현재 이 상황에서는 시퀀스로 변환하는 것이 별다른 효과를 보지 못하지 않나요..?
성능적으로 큰 이점이 없을 것 같은데 굳이 시퀀스를 사용하시는 이유가 있을까요??

Reply:
재녕님이 말씀하신대로 단순히 map 호출을 두 번 하고 있기 때문에, 시퀀스를 사용한다고 해서 별다른 효과를 보지 못할 수도 있습니다.
하지만 대부분 처리 단계가 두 단계 이상인 경우에는 시퀀스를 사용하는 습관을 들여주시면 좋습니다.
(자주 활용을 해봐야 머리에 잘 남으니까요)